requirements.txt

```diff
@@ -1,5 +1,5 @@
 git-lfs
-natcap.invest
+git+https://github.com/emlys/invest.git@task/777
```

Comment:
I believe the changes are now in `natcap.invest`. Should we switch this back?

Reply:
They haven't been released yet, but I changed it to point to `natcap/invest@main`.

Reply:
Oh, right. I guess that's what I was thinking.